frontend/app-development/features/textEditor/TextEditor.test.tsx

```diff
@@ -1,9 +1,43 @@
 import { renderWithProviders } from '../../test/testUtils';
 import { APP_DEVELOPMENT_BASENAME } from 'app-shared/constants';
-import { screen } from '@testing-library/react';
+import { act, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
 import React from 'react';
 import { TextEditor } from './TextEditor';
 import { textMock } from '../../../testing/mocks/i18nMock';
+import { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
+import userEvent from '@testing-library/user-event';
+
+const user = userEvent.setup();
```

Comment:
According to the documentation we should not share userEvent.setup between tests like this. 

"We recommend invoking [userEvent.setup()](https://testing-library.com/docs/user-event/setup) before the component is rendered. This can be done in the test itself, or by using a setup function

**Link to the docs:**
https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent

Reply:
Done!